BreastDx-01-0003/AllergyIntolerance/allergy_intolerance.breastdx.json

```diff
@@ -0,0 +1,34 @@
+// allergy_intolerance.example.json
+// id: breastdx
```

Comment:
This is the ID for the allergy_intolerance - should be set to random as it's unlikely to be referenced elsewhere